lib/jsdom/living/attributes.js

```diff
@@ -1,6 +1,17 @@
 "use strict";
 const DOMException = require("../web-idl/DOMException");
 const defineGetter = require("../utils").defineGetter;
+const idlUtils = require("./generated/utils");
+const attrGenerated = require("./generated/Attr");
+const changeAttributeImpl = require("./attributes/Attr-impl").changeAttributeImpl;
+
+function attrImpl(attrWrapper) {
+  return idlUtils.implForWrapper(attrWrapper, "Attr");
+}
+
+function isAttr(obj) {
+  return Boolean(obj && obj.constructor && obj.constructor.name === "Attr"); // TODO branding
```

Comment:
Oh nice. Can you have Attr.js export a brand check for me in all generated classes?